### DISCUSSION TITLE
Any plan for support for DEBERTA-V2

### DISCUSSION THREAD

**Discussion by lordofthejars** [tjake/Jlama#52] [General]
Hi I'd love to use one model of Meta based on DEBERTA-V2 model from Microsoft. When I run the model on jlama I got a `: java.lang.IllegalArgumentException: No enum constant com.github.tjake.jlama.model.ModelSupport.ModelType.DEBERTA-V2`.

Is there any plan to provide support for this kind of models?

Thank you very much.

**Comment by tjake** [comment]
Sure I opened an issue for this #53

I also need to add phi support
